.travis.yml

```diff
@@ -159,9 +159,8 @@ script:
   - export CMAKE_ARGS="-DLUA_LIBRARY=${LUALIBRARY} -DLUA_INCLUDE_DIR=${LUAINCLUDE} -D OVERRIDE_VERSION=$AWESOME_VERSION -DSTRICT_TESTS=true"
   - |
     if [ "$DO_COVERAGE" = "codecov" ]; then
-      export CXXFLAGS="-fprofile-arcs -ftest-coverage"
       export CFLAGS="-fprofile-arcs -ftest-coverage"
-      export LDFLAFS="-fprofile-arcs -ftest-coverage"
+      export LDFLAGS="-fprofile-arcs"
```

Comment:
> Please improve the commit message.

Ok.

It looks like we might not need the flag at all - it had a typo before, but somewhere I've read that it should use -fprofile-arcs for linking, but could not find it later again.